x-pack/plugins/observability/public/utils/observability_data_views/observability_data_views.ts

```diff
@@ -49,7 +49,7 @@ const appToPatternMap: Record<AppDataType, string> = {
   synthetics: '(synthetics-data-view)*',
   apm: 'apm-*',
   ux: '(rum-data-view)*',
-  infra_logs: '',
+  infra_logs: '(infra-logs-data-view)*',
```

Comment:
@shahzad31 Do you remember why these patterns are prepended? I remember trying to remove the logs one and that lead to errors, so I put this back, but like Felix says it will potentially lead to negative side effects. As far as I can tell the rum and synthetics ones aren’t “real” either, but they will become part of a real and persisted data view. If you don’t remember I can trace this back through again and make sure this prepended part can be optional and not cause runtime errors.

Reply:
There is no purpose, it's just a way to identify that if we have many data views so we know these are being used in the exploratory view.

Once we have the ability to name data views, I think we can get rid of these. For querying perspective these doesn't makes a difference.

Reply:
> For querying perspective these doesn't makes a difference.

What if a user has indices that match the pattern?

Reply:
@shahzad31 

> what issue you faced when you removed it?

If you leave `infra_logs` as an empty string the following error gets thrown:

![Screenshot 2022-03-18 at 18 28 20](https://user-images.githubusercontent.com/471693/159062736-3143ff12-7655-40cc-b734-976ba8b8f52c.png)

The reason it gets thrown is because of this request:

![Screenshot 2022-03-18 at 18 30 55](https://user-images.githubusercontent.com/471693/159062818-ce6f273c-0af1-48b3-afc2-6b9b98241188.png)

The pattern query params are `,remote_cluster:filebeat-*,remote_cluster:kibana_sample_data_logs*,filebeat-*,kibana_sample_data_logs*` (note the comma at the start).

I assume it's a pretty easy fix, stop the comma being prepended when there isn't a value set but I didn't deep dive on this one.

I can file an issue though, along with the series names issue.